_posts/2015-11-13-svg-sprites-angular.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: Svg sprites
+permalink: /svg-sprites
+tags: angular, svg, sprite
+author_id: sinisag
+category: webapp
+---
+
+(CSS) Sprite is a method, where we combine multiple small images or graphic in to one large image and access the needed graphic by specifying 'coordinates' on the big image.
```

Comment:
We could add a link to https://en.wikipedia.org/w/index.php?title=Sprite_(computer_graphics)&redirect=no#Sprites_by_CSS